desc/plotting.py

```diff
@@ -626,8 +630,28 @@ def plot_2d(
     plot_axes = _get_plot_axes(grid)
     if len(plot_axes) != 2:
         return ValueError(colored("Grid must be 2D", "red"))
-
-    data, label = _compute(eq, name, grid, kwargs.pop("component", None))
+    component = kwargs.pop("component", None)
+    if name != "Bn":
```

Comment:
Can we change the name to `"B*n"` to better match our existing format for other compute quantities? For example, we already have stuff like `"B*grad(|B|)"`